posthog/hogql_queries/web_analytics/stats_table_pre_aggregated.py

```diff
@@ -150,7 +155,10 @@ def _get_order_by(self):
                 column = "context.columns.visitors"
             elif field == WebAnalyticsOrderByFields.VIEWS:
                 column = "context.columns.views"
-            elif field == WebAnalyticsOrderByFields.BOUNCE_RATE:
+            elif field == WebAnalyticsOrderByFields.BOUNCE_RATE and self.runner.query.breakdownBy in [
+                WebStatsBreakdown.INITIAL_PAGE,
+                WebStatsBreakdown.PAGE,
+            ]:
```

Comment:
Unrelated to the filter but it fixes a bug when sorting by bounce_rate on other tiles.